demo-app/src/main/AndroidManifest.xml

```diff
@@ -29,6 +29,7 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:networkSecurityConfig="@xml/network_security_config"
```

Comment:
Why are we changing network security config for noise cancelation?

Reply:
I was probably investigating some issue within the demo app. That's for Proxyman, which won't work by default unless you install the certificate into your phone